docker/quick-start/README.md

```diff
@@ -0,0 +1,72 @@
+# Quickstart a test Bacalhau node in containers
+
+```
+docker build -t ipfs --target ipfs .
+docker build -t bacalhau --target bacalhau .
+
+# TODO: run container as readonly fs
+docker run \
+    -dit \
+    --name bacalhau-ipfs \
+    --restart always \
+    --net host \
+    --volume bacalhau-ipfs:/data \
+        ipfs
+
+
+docker logs -f bacalhau-ipfs
+
+ctrl-c
+
+docker exec -it bacalhau-ipfs  ipfs cat /ipfs/QmQPeNsJPyVWPFDVHb77w8G42Fvo15z4bG2X8D2GhfbSXc/readme
+
+#export IPFS_CONNECT=$(docker exec -it bacalhau-ipfs  ipfs id | grep #tcp | grep 127.0.0.1 | sed 's/4001/5001/' | sed 's/^[ \t]*//' | sed #'s/,//')
+
+#TODO: 02:07:28.327 | DBG bacalhau/serve.go:284 > libp2p connecting to: [/ip4/35.245.115.191/tcp/1235/p2p/QmdZQ7ZbhnvWY1J12XYKGHApJ6aufKyLNSvf8jZBrBaAVL /ip4/35.245.61.251/tcp/1235/p2p/QmXaXu9N5GNetatsvwnTfQqNtSeKAD6uCmarbh3LMRYAcF /ip4/35.245.251.239/tcp/1235/p2p/QmYgxZiySj3MRkwLSL4X2MF5F9f2PMhAE3LV49XkfNL1o3]
+': failed to parse multiaddr "\"/ip4/127.0.0.1/tcp/5001/p2p/12D3KooWDN19JJkptojgerCwHYYJcZcz7whuMGsm7Dqd86kutGb3\"\r": must begin with /b3"
+
+# TODO: run container as readonly fs
+docker run \
+    -dit \
+    --name bacalhau \
+    --restart always \
+    --env IPFS_CONNECT=$( docker exec -it bacalhau-ipfs ipfs id --format="/ip4/127.0.0.1/tcp/5001/p2p/<id>") \
+    --net host \
+    --volume bacalhau-data:/data \
+    --volume /run/docker.sock:/run/docker.sock \
+    --volume /tmp:/tmp \
+    --volume $(which docker):$(which docker) \
+        bacalhau
+
+
+docker logs -f bacalhau
+
+ctrl-c
+
+# To see that bacalhau is running your docker job locally, you can run 'docker events` in another terminal, and you should see the container 'create', 'start', 'die', and 'destroy' events for a container with a name starting with 'bacalhau'
+
+docker exec -it \
+    --env BACALHAU_API_HOST=127.0.0.1 \
+    --env BACALHAU_API_PORT=1234 \
+    bacalhau \
+     bacalhau docker run ubuntu echo hello
+
+# Interestingly, because your bacalhau node is a part of the bootstrap-cluster, you can get info about your job directly from your node, or via the <<<insert dns..name>>>
```

Comment:
this actually makes me uncomfortable - I _think_ I'd rather start with a local, not joined to the global bacalhau network - so as not to worry new users / their IP management team

Reply:
in the slack thread, @philwinder suggested quickstart would be for a user adding a temporary node to the global bacalhau network, and then the 'make a totally safe temporary? private bacalhau network' would be a separate doc.

This is a more realistic goal for my current knowledge :) and I can start on the other in a follow on PR